icssc-projects/peterportal-client
# User auth

## Description

Integrated passport.js to our project. Created a user endpoint for account interface (login/logout).

## Screenshots
Added back the profile icon.
![image](https://user-images.githubusercontent.com/44650208/113462003-c6587300-93d3-11eb-9edc-6e48b9bcb820.png)
No shows their name in reviews.
![image](https://user-images.githubusercontent.com/44650208/113462031-e720c880-93d3-11eb-8478-3f51347b50aa.png)

## Steps to verify/test this change:
1. Sign in with google.
2. Check that you can see your profile picture and name on the profile icon.
3. Write a review and see if your name shows up.

## Todos:
- [ ] Use a session store with mongo or sql.
